src/DependencyInjection/Configuration.php

```diff
@@ -145,6 +145,16 @@ public function getConfigTreeBuilder(): TreeBuilder
                             ->defaultValue([])
                             ->prototype('scalar')->end()
                         ->end()
+                        ->arrayNode('frontend_stylesheets')
```

Comment:
There is no need to add a new option, please use the above. It is only used for the frontend.

Reply:
Or even better, IMO this template will be normally replaced for the user because he will be using its own design without bootstrap. 

I think we should deprecate / remove this config and place the css without being a config option.

Reply:
Yeah i guess it'll be replaced too, but if we don't provide an Frontend css, when the user I install the page bundle the standard layout will be ugly

Reply:
No, it wont, by default we will provide frontend.css. but it doesn't need to be configurable, it can be just placed on the template.

Reply:
Hmmm ok, I'll try that tomorrow :)